Cargo.toml

```
@@ -6,14 +6,27 @@ license = "MIT"
 authors = ["Gear Technologies"]
 
 [dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], rev = "d4552434" }
+gstd = { git = "https://github.com/gear-tech/gear.git", features = [
+    "debug",
+], branch = "stable" }
```

I like it more when one feature is in one string, but it doesn't matter.
```suggestion
gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], branch = "stable" }
```

For example, look at some other projects.

https://github.com/rust-lang/cargo/blob/master/Cargo.toml#L29
https://github.com/tokio-rs/tokio/blob/master/tokio/Cargo.toml#L117